pymeasure/instruments/spellmanhv/spellmanXRV.py

```
@@ -531,10 +531,12 @@ def set_scaling(self):
                                     },
         )
 
-    def reset_hv_on_timer(self):
+    def reset_hv_on_timer(self) -> None:
+        """Reset the HV on hours counter."""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF002)